### DISCUSSION TITLE
Application Loader seems to be deprecated... Can't find it on OSX. How do you do to upload your build now ?

### DISCUSSION THREAD

**Discussion by stephanberger4** [airsdk/Adobe-Runtime-Support#2438] [Q&A] [answered]
I use Adobe Animate to publish my ipa build. Then I use Application Loader to upload my build to App store...
But my applicaiton loader app have be uninstalled (I use MacinCloud as I don't have a mac). And I can't find the app anywhere. 

Was it deprecated ? If so, how do you upload your app now ?

**Comment by marchbold** [comment, marked as answer]
Yes Application Loader was deprecated a while ago. 

You should use Transporter now:

https://apps.apple.com/au/app/transporter/id1450874784?mt=12

**Reply by stephanberger4** [reply]
Thx ! Weirdly, I was still using it 2 weeks ago ! I will try it as soon as my macincloud will reboot (froze...)

**Reply by marchbold** [reply]
Yeah right, must have had an old copy that still works. I don't think it's been included with Xcode for several versions now but honestly don't remember when we switched over.
